tests/configurable_cluster.py

```diff
@@ -106,7 +106,7 @@ def tear_down(self):
             # is resolved https://github.com/prestodb/presto-admin/issues/226
             script = """
             sudo service presto stop
-            sudo rpm -e presto-server-rpm
+            sudo rpm -e starburst-presto-server-rpm
```

Comment:
Probably yes. But the tests in open source was already broken because whatever OS it runs on doesnt support all the python dependency version we updated.